pyramid_mixpanel/consumer.py

```diff
@@ -52,7 +50,16 @@ def flush(self, *args, **kwargs) -> None:
         try:
             super(PoliteBufferedConsumer, self).flush(*args, **kwargs)
         except URLError:
-            logger.exception("It seems like Mixpanel is down.", exc_info=True)
+            if importlib.util.find_spec("structlog"):  # type: ignore
+                import structlog
+
+                logger = structlog.get_logger(__name__)
+                logger.exception("It seems like Mixpanel is down.", exc_info=True)
+            else:
+                import logging
+
+                logger = logging.getLogger(__name__)
+                logger.exception("It seems like Mixpanel is down.", exc_info=True)
```

Comment:
I gave up to quickly with getting `request` to `PoliteBufferedConsumer` to do it like in https://github.com/niteoweb/pyramid_heroku/blob/cdf99f5f3cba738872fd9b705b05bf0e700fa099/pyramid_heroku/herokuapp_access.py#L39. 

Will rewrite.